core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationSummaryInfo.scala

```diff
@@ -73,6 +80,9 @@ class AppSummaryInfoBaseProvider extends AppInfoPropertyGetter
   override def getJvmGCFractions: Seq[Double] = Seq()
   override def getSpilledMetrics: Seq[Long] = Seq()
   override def getRapidsJars: Seq[String] = Seq()
+  override def isParquetOrOrc: Boolean = false
```

Comment:
ORC filecache support is not merged yet, unclear if this will be 23.06 or maybe 23.08.  I'd also recommend changing this to 'hasFilecacheSupportedFormat` or something similar to avoid making this confusing if/when we support caching other formats.

Reply:
Updated to `hasFileCacheSupportedFormat` in latest commit

Reply:
Removed this function in latest commit